Formula/haskell-language-server.rb

```diff
@@ -1,5 +1,5 @@
 class HaskellLanguageServer < Formula
-  desc "Integration point for ghcide and haskell-ide-engine. One IDE to rule them all"
+  desc "HLS - binary built for GHC 8.10"
```

Comment:
Acronyms aren't really a useful description here, I think.

Reply:
Sure, I tried with full name first, but test bot failed because it was repeating of the formula name (BTW local `brew audit/style` do not report this). Wanted to change not to repeat exactly same text in each HLS formula...

Reply:
Why not just `Language Server for ghc #{Formula["ghc"].version}`?